Assets/Scripts/PlayerScripts/HoverSelect.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HoverSelect : MonoBehaviour
+{
+    public Button card; //in Unity editor set this as the button this script is attached to.
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    public void SelectButton()
```

Comment:
Is SelectButton() used anywhere? I can't find anything calling it.

Actually, I can't find any HoverSelect.cs usage. Am I missing something obv, or can we remove it?